saleor/webhook/payloads.py

```diff
@@ -579,15 +579,19 @@ def generate_collection_payload(
 def serialize_product_channel_listing_payload(channel_listings):
     serializer = PayloadSerializer()
     fields = (
-        "publication_date",
+        "published_at",
         "id_published",
         "visible_in_listings",
-        "available_for_purchase",
+        "available_for_purchase_at",
     )
     channel_listing_payload = serializer.serialize(
         channel_listings,
         fields=fields,
-        extra_dict_data={"channel_slug": lambda pch: pch.channel.slug},
+        extra_dict_data={
+            "channel_slug": lambda pch: pch.channel.slug,
+            "publication_date": lambda pch: pch.published_at,
+            "available_for_purchase": lambda pch: pch.available_for_purchase_at,
```

Comment:
Maybe we could put some line comments here to mention that these are deprecated?